app/src/main/java/org/mozilla/fenix/IntentReceiverActivity.kt

```diff
@@ -48,7 +41,7 @@ class IntentReceiverActivity : Activity() {
 
             val intentProcessors = listOf(
                 components.utils.customTabIntentProcessor,
-                if (isPrivate) components.utils.privateIntentProcessor else components.utils.intentProcessor
```

Comment:
@sblatz doesn't this undo private browsing when enabled?

Reply:
It doesn't look like this was added as part of the private launcher from what I can tell: https://github.com/mozilla-mobile/fenix/commit/2e2bac4ccdc24e5dc095566023c50cf1ce1d2773#diff-842c3d4ac654e221b378bcb5855d8cc1L46